source/agora/node/Config.d

```diff
@@ -100,11 +100,11 @@ public struct Config
     public RegistryConfig registry;
 
     /// The list of IPs for use with network discovery
-    public @Optional immutable string[] network;
+    public @Optional immutable Address[] network;
 
     /// The list of DNS FQDN seeds for use with network discovery
     @Name("dns")
-    public @Optional immutable string[] dns_seeds;
+    public @Optional immutable Address[] dns_seeds;
```

Comment:
We should nuke this field, eventually, so I'm less fussed about it being changed to `Address` if it simplifies your life.